sortingArray/sortArray.js

```diff
@@ -0,0 +1,13 @@
+export const sortArrays = arr => {
```

Comment:
Modify this function to take two args. One is `array` and the other to mention it as `asc` or `dsc`.
so this method should be something like below ...

sortArrays([1,5,2], "asc" ); //  [1,2,5]
sortArrays([1,5,2], "dsc" ); //  [5,2,1]

Reply:
fixed